pkgs/by-name/im/imake/package.nix

```diff
@@ -64,14 +64,15 @@ stdenv.mkDerivation (finalAttrs: {
     CFLAGS = "-DIMAKE_COMPILETIME_CPP='\"${
       if stdenv.hostPlatform.isDarwin then "${tradcpp}/bin/cpp" else "gcc"
     }\"'";
+    inherit tradcpp;
   };
 
   preInstall = ''
     mkdir -p $out/lib/X11/config
     ln -s ${xorg-cf-files}/lib/X11/config/* $out/lib/X11/config
   '';
 
-  inherit tradcpp xorg-cf-files;
+  inherit xorg-cf-files;
```

Comment:
Does the fact that `xorg-cf-files` does not need to be adjusted mean that it's actually unused, at least via `inherit`?